openmmtools/alchemy.py

```diff
@@ -860,6 +886,84 @@ def get_energy_components(cls, alchemical_system, alchemical_state, positions, p
 
         return energy_components
 
+    def replace_reaction_field(self, reference_system):
+        """Replace reaction-field electrostatics with Custom*Force terms to ensure c_rf = 0.
+
+        A deep copy of the system is made.
+
+        If reaction field electrostatics is in use, this will add a CustomNonbondedForce and CustomBondForce to the System
+        for each NonbondedForce that utilizes CutoffPeriodic.
+
+        Note that the resulting System object can NOT be fed to `create_alchemical_system` since the CustomNonbondedForce
+        will not be recognized and re-coded.
+
+        Parameters
+        ----------
+        reference_system : simtk.openmm.System
+            The system to use as a reference for the creation of the
+            alchemical system. This will not be modified.
+        switch_width : simtk.unit.Quantity with units compatible with distance, optional, default = 1.0 * angstroms
```

Comment:
Whoops! This was moved to the `AlchemicalFactory` constructor and should be removed from this docstring.